capellambse/model/common/accessors.py

```diff
@@ -390,6 +390,32 @@ def __get__(self, obj, objtype=None):
         ]
         return self._make_list(obj, elems)
 
+    def __set__(self, obj, value):
+        is_it = isinstance(value, cabc.Iterable) and not isinstance(value, str)
```

Comment:
You named the boolean variable `is_it` which is short for `is_iterable` I suppose. Really verbose would be `is_iterable_non_string` which reminds me of the [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation). Something invaluable I learned during my studys (/s). There is enough space for the verbose form, so why shortening?